docs/fiddles/features/keyboard-shortcuts/interception-from-main/main.js

```diff
@@ -0,0 +1,13 @@
+const { app, BrowserWindow } = require('electron')
+
+app.whenReady().then(() => {
+  const win = new BrowserWindow({ width: 800, height: 600, webPreferences: { nodeIntegration: true } })
```

Comment:
Weird linting issue on this line here?

Reply:
No issues on my side.
I was receiving the following error related to TypeScript:

```
✔ Electron API generated in "/electron" took 5.2s
⠋ Checking argvTypeError: Cannot read property 'toLowerCase' of undefined
    at Object.exports.paramify (node_modules/@electron/typescript-definitions/dist/utils.js:199:23)
    at node_modules/@electron/typescript-definitions/dist/utils.js:309:30
    at Array.map (<anonymous>)
    at Object.exports.genMethodString (node_modules/@electron/typescript-definitions/dist/utils.js:283:10)
    at node_modules/@electron/typescript-definitions/dist/utils.js:307:33
    at Array.map (<anonymous>)
    at Object.exports.genMethodString (node_modules/@electron/typescript-definitions/dist/utils.js:283:10)
    at addMethod (node_modules/@electron/typescript-definitions/dist/module-declaration.js:154:35)
    at node_modules/@electron/typescript-definitions/dist/module-declaration.js:177:90
    at Array.forEach (<anonymous>)
```

but `rm -rf node_modules && npm i` did the trick.